examples/tutorials/forced_alignment_for_multilingual_data_tutorial.py

```diff
@@ -0,0 +1,387 @@
+"""
+Forced alignment for multilingual data
+======================================
+
+**Author**: `Xiaohui Zhang <xiaohuizhang@meta.com>`__
+
+This tutorial shows how to compute forced alignments for speech data
+from multiple languages using ``torchaudio``\ ’s CTC forced alignment
+API described in `“CTC forced alignment
+tutorial” <https://pytorch.org/audio/stable/tutorials/forced_alignment_tutorial.html>`__
+and the multilingual Wav2vec2 model proposed in the paper `“Scaling
+Speech Technology to 1,000+
+Languages” <https://research.facebook.com/publications/scaling-speech-technology-to-1000-languages/>`__.
+The model was trained on 23K of audio data from 1100+ languages using
+the `“uroman vocabulary” <https://www.isi.edu/~ulf/uroman.html>`__
+as targets.
+
+"""
+
+import torch
+import torchaudio
+
+print(torch.__version__)
+print(torchaudio.__version__)
+
+
+device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+print(device)
+
+
+try:
+    from torchaudio.functional import forced_align
+except ModuleNotFoundError:
+    print(
+        "Failed to import the forced alignment API. "
+        "Please install torchaudio nightly builds. "
+        "Please refer to https://pytorch.org/get-started/locally "
+        "for instructions to install a nightly build."
+    )
+    raise
+
+######################################################################
+# I. Preparation
```

Comment:
We don't use roman numerals. Please use arabic numerals or just simply remove them.

Reply:
sure